### DISCUSSION TITLE
Sonoff D1 flashing: Can't connect

### DISCUSSION THREAD

**Discussion by jenshenk** [arendst/Tasmota#12231] [Q&A] [answered]
Hi!
After serial flashing around 10 Sonoff S20s and a Sonoff RF bridge I'd consider myself an advanced tasmotizing beginner and thought I'll give the D1 a shot, yet somehow I don't have success. 

What I managed:
- connect it to the serial flasher without pulling GPIO 0 down: device boots, blue led blinks (this just to check whether it's a cable or some other electrical issue). 
- connect to the serial flasher WITH gpio 0 pulled to ground: no blue led blinking, I assume device is now in flashing mode
- I connect TX and RX and click tasmotize in tazmotizer (as I've done a lot of times with the S20s).... yet it times out every time :(

Any ideas to what I'm doing wrong? Points I was wondering about:
- I now connect 3v and ground to the pins on the backside, does it make a difference whether it's connected to that pins compared to the "holes" next to the tx and rx? Wouldn't make sense, but who knows
- Maybe TX and RX are mislabeled? I already tried to change them around, but maybe they are somewhere completely different on the board? My D1 board is V 1.1

Very grateful for any tips as I was really looking forward to integrate a couple of D1s into my setup. 
TIA!

Edit: (tinkering around to rule out some potential reasons) 
- just for the heck of it I connected the pins all in one row, still no success
- I had a second sonoff rf bridge lying around that I bought as a backup and that wasn't flashed yet... with the same cables and serial flasher I flashed it without any problems... so it can't be the cables, nor the serial flasher itself... Really at a loss here :/

**Comment by ascillato2** [comment, marked as answer]
Please, address this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4). The chat is a better and more dynamic channel for helping you. Thanks.

---------------------------

### Support Information 
See [Docs](https://tasmota.github.io/docs/) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
See [Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)

**Comment by jenshenk** [comment]
Thanks for pointing me there! In the meantime I already orderered the Sonoff remote for just 6 bucks so I'll be able going the DIY route soon... Yet, next to the necessity my curiosity why it did not not work via serial is still too big to just forget about it :)
